### ISSUE TITLE
rabbitmq.list_users is missing the last user

### ISSUE THREAD

**Issue by shawnbutts** [saltstack/salt#25000]
```yaml
           Salt: 2015.5.2
         Python: 2.6.9 (unknown, Apr  1 2015, 18:16:00)
         Jinja2: 2.7.2
       M2Crypto: 0.21.1
 msgpack-python: 0.4.6
   msgpack-pure: Not Installed
       pycrypto: 2.6.1
        libnacl: Not Installed
         PyYAML: 3.10
          ioflo: Not Installed
          PyZMQ: 14.3.1
           RAET: Not Installed
            ZMQ: 3.2.5
           Mako: Not Installed
```

```yaml
# salt-call rabbitmq.list_users
local:
    ----------
    admin:
    user1:
    user2:
    user3:
    user4:
    user5:
    user6:


# rabbitmqctl list_users
Listing users ...
admin   [administrator]
user1     [monitoring,user]
user2     [monitoring,user]
user3     [monitoring,user]
user4     [monitoring,user]
user5     [monitoring,user]
user6     [monitoring,user]
user7 [monitoring]
```

This breaks a number of states for me.

**Comment by shawnbutts** [comment]
The workaround I've found is to manually add a user

**Comment by jfindlay** [comment]
@shawnbutts, thanks for the report.

**Comment by simonbp** [comment]
This appears to be fixed as of 2015.8.1 (Beryllium)

```
# rabbitmqctl list_users
Listing users ...
admin	[administrator]
app_user	[]
...done.

# salt-call rabbitmq.list_users
stdin: is not a tty
[INFO    ] Executing command 'rabbitmqctl list_users' as user 'root' in directory '/root'
local:
    ----------
    admin:
    app_user:
```

**Comment by jfindlay** [comment]
Since this issue seems to be fixed, I'm going to close it.  Thanks @shawnbutts, @simonbp.
